ci/prow/make_config.go

```diff
@@ -966,7 +997,7 @@ func collectMetaData(periodicJob yaml.MapSlice) {
 			releaseVersion := ""
 			for _, item := range jobConfig {
 				switch item.Key {
-				case "continuous", "dot-release", "auto-release", "performance",
+				case "continuous", "continuous-go114", "dot-release", "auto-release", "performance",
```

Comment:
This change can be removed if you use custom jobs?

Reply:
Removed.